tests/buffer.spec.ts

```diff
@@ -27,7 +27,7 @@ describe("buffer", () => {
     expect<Buffer>(Buffer.alloc(10)).toHaveLength(10);
     let buff = Buffer.alloc(100);
     for (let i = 0; i < buff.length; i++) expect<u8>(buff[i]).toBe(0);
-    expect<ArrayBuffer>(buff.buffer).not.toBeNull();
+    expect<ArrayBuffer | null>(buff.buffer).not.toBeNull();
```

Comment:
This looks like it can't happen

Reply:
Remember that we call the `__alloc` function and set the property manually. This is just a sanity check to make sure that the `.buffer` property actually points to something.

Chances are it's something we won't forget, but I've forgotten way simpler things.